src/main/java/uk/gov/hmcts/ccd/data/definition/DefaultCaseDefinitionRepository.java

```diff
@@ -129,28 +135,26 @@ public UserRole getUserRoleClassifications(String userRole) {
             final HttpEntity requestEntity = new HttpEntity<CaseType>(securityUtils.authorizationHeaders());
             final Map<String, String> queryParams = new HashMap<>();
             queryParams.put("roles", StringUtils.join(userRoles, ","));
-            return Arrays.asList(restTemplate.exchange(applicationParams.userRolesClassificationsURL(), HttpMethod.GET, requestEntity, UserRole[].class, queryParams).getBody());
+            return Arrays.asList(ofNullable(getClassificationForUserRoleList(requestEntity, queryParams)).orElseThrow(() -> NULL_RESPONSE_EX).getBody());
         } catch (Exception e) {
-            LOG.warn("Error while retrieving classification for user roles {} because of ", userRoles, e);
-            throw new ServiceException("Error while retrieving classification for user roles " + userRoles + " because of " + e.getMessage());
+            LOG.warn("Error while retrieving classification for user roles {} because of: ", userRoles, e);
+            throw new ServiceException("Error while retrieving classification for user roles " + userRoles + " because of: " + e.getMessage());
         }
```

Comment:
Looks like you could also use public ServiceException(final String message, final Exception cause)
That way, you do not throw away the stack trace.  What do you think?